docs/DeveloperGuide.md

```diff
@@ -2,6 +2,15 @@
 layout: page
 title: Developer Guide
 ---
+
+LinkedOUT is the only application that any experienced recruiter needs. LinkedOUT allows recruiters to keep track of many applicants, and the job they applied for. You can store their contact details, skills and the round of their application, all in one place.
+
+LinkedOUT helps recruiters manage the multiple job applications they may receive on a daily basis. With many applications, it may be difficult to keep track of each applicant and which round they are currently at.
```

Comment:
do you think we should use 'application round' , 'round of application', like some phrase to make it clear what round is since we are not using stage
```suggestion
LinkedOUT helps recruiters manage the multiple job applications they may receive on a daily basis. With many applications, it may be difficult to keep track of each applicant and which application round they are currently at.
```

Reply:
Yup thanks for the suggestion!